src/NLog.Targets.Syslog/MessageSend/MessageTransmitter.cs

```diff
@@ -38,7 +38,9 @@ public static MessageTransmitter FromConfig(MessageTransmitterConfig messageTran
         protected MessageTransmitter(string server, int port)
         {
             Server = server;
-            IpAddress = Dns.GetHostAddresses(server).FirstOrDefault()?.ToString();
+            var hostAddresses = Dns.GetHostAddressesAsync(server);
+            hostAddresses.Wait();
+            IpAddress = hostAddresses.Result.FirstOrDefault()?.ToString();
```

Comment:
Calling a blocking Wait (or Result) on an async method is useless: please keep the old code

Reply:
Fixed